app/invocation/invocation.css

```diff
@@ -55,6 +55,23 @@
   background-color: #eee;
 }
 
+.header {
+  display: flex;
+}
+
+.header .button {
+  flex-shrink: 1;
```

Comment:
i'd just make this `flex-grow: 0` and instead of `justify-content: right`, use `justify-content: space-between` on .header

you can also remove the ` flex-grow: 9` below

that way we don't have any arbitrary 9-1 ratios